graphql/src/index.ts

```diff
@@ -265,6 +274,22 @@ const resolvers = {
     accessibility({ venue }: any, args: any, context: any, info: any) {
       return venue.accessibility;
     },
+    orderedByAccessibilityShortcoming(
+      { venue }: any,
+      args: any,
+      context: any,
+      info: any
+    ) {
+      if (args.profile) {
+        const shortcoming = venue.accessibility.shortcomings.filter(
+          (shortcoming) => shortcoming.profile == args.profile
```

Comment:
How about triple equation? I guess it does not matter much here. 🤔

Reply:
This could be simplified to a lot shorter version:

```javascript
      if (args.profile) {
        const shortcoming = venue.accessibility.shortcomings.filter(
          (shortcoming) => shortcoming.profile == args.profile
        );
        if (shortcoming && shortcoming.length > 0) {
          return shortcoming[0];
        }
      }
      return undefined;
```
→
```javascript
      return venue.accessibility?.shortcomings?.filter(
          (shortcoming) => shortcoming.profile == args.profile
        )[0];
```
but is it as obvious from this code that undefined is returned on purpose when no shortcoming is found for the profile?
I don't think so and that's why I left the code in a longer form to make this intention explicit.

An alternative
```javascript
      return args.profile ? venue.accessibility?.shortcomings?.filter(
          (shortcoming) => shortcoming.profile == args.profile
        )[0] : undefined;
```
would be a bit more explicit but I don't like the implicit returning of undefined in the case when the filtered shortcomings list is empty i.e. `[][0] === undefined`.

How do you see this with these additions?